File: app/src/main/java/com/example/i18n/AppStrings.kt
package com.example.i18n

object AppStrings {
    fun appTitle(lang: AppLanguage) = "Prends & Rends"
    fun appSubtitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Suivi des prêts & emprunts"
        AppLanguage.EN -> "Loan & borrow tracker"
        AppLanguage.HA -> "Maikula da bashi da aro"
    }

    // Tabs
    fun tabAll(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Tous"
        AppLanguage.EN -> "All"
        AppLanguage.HA -> "Dukkansu"
    }
    fun tabLent(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Prêtés"
        AppLanguage.EN -> "Lent"
        AppLanguage.HA -> "Aka bayar"
    }
    fun tabBorrowed(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Empruntés"
        AppLanguage.EN -> "Borrowed"
        AppLanguage.HA -> "Aka aro"
    }

    // Status filters
    fun filterAll(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Tous les statuts"
        AppLanguage.EN -> "All statuses"
        AppLanguage.HA -> "Dukan matsayi"
    }
    fun filterActive(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "En cours"
        AppLanguage.EN -> "Active"
        AppLanguage.HA -> "Yana gudana"
    }
    fun filterOverdue(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "En retard"
        AppLanguage.EN -> "Overdue"
        AppLanguage.HA -> "Ya wuce lokaci"
    }
    fun filterReturned(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Rendu"
        AppLanguage.EN -> "Returned"
        AppLanguage.HA -> "An mayar"
    }

    // Dashboard
    fun newLoanBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Nouveau prêt"
        AppLanguage.EN -> "New loan"
        AppLanguage.HA -> "Sabon bashi"
    }
    fun searchPlaceholder(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Rechercher un objet, contact..."
        AppLanguage.EN -> "Search item, contact..."
        AppLanguage.HA -> "Nemi abu, lamba..."
    }
    fun statsLent(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Prêtés"
        AppLanguage.EN -> "Lent"
        AppLanguage.HA -> "Aka bayar"
    }
    fun statsBorrowed(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Empruntés"
        AppLanguage.EN -> "Borrowed"
        AppLanguage.HA -> "Aka aro"
    }
    fun statsOverdue(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "En retard"
        AppLanguage.EN -> "Overdue"
        AppLanguage.HA -> "Aka makara"
    }
    fun statsReturned(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Rendus"
        AppLanguage.EN -> "Returned"
        AppLanguage.HA -> "Aka mayar"
    }
    fun emptyTitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Aucun prêt enregistré"
        AppLanguage.EN -> "No loans recorded"
        AppLanguage.HA -> "Babu bashi a ajiye"
    }
    fun emptySubtitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Gardez une trace fluide de ce que vous prêtez ou empruntez."
        AppLanguage.EN -> "Keep smooth track of items and money lent or borrowed."
        AppLanguage.HA -> "Kula da abin da kake bayarwa ko aro cikin sauki."
    }

    // Form
    fun createTitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Nouveau prêt"
        AppLanguage.EN -> "New Loan"
        AppLanguage.HA -> "Sabon Bashi"
    }
    fun editTitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Modifier le prêt"
        AppLanguage.EN -> "Edit Loan"
        AppLanguage.HA -> "Gyara Bashi"
    }
    fun typeHeader(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Type de prêt"
        AppLanguage.EN -> "Loan type"
        AppLanguage.HA -> "Nau'in bashi"
    }
    fun typeLent(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "J'ai prêté"
        AppLanguage.EN -> "I lent"
        AppLanguage.HA -> "Na bayar"
    }
    fun typeBorrowed(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "J'ai emprunté"
        AppLanguage.EN -> "I borrowed"
        AppLanguage.HA -> "Na aro"
    }
    fun typeLentSub(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "À un proche"
        AppLanguage.EN -> "To a contact"
        AppLanguage.HA -> "Ga wani"
    }
    fun typeBorrowedSub(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "D'un proche"
        AppLanguage.EN -> "From a contact"
        AppLanguage.HA -> "Daga wani"
    }
    fun titleLabel(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Objet ou Motif du prêt *"
        AppLanguage.EN -> "Item or Reason *"
        AppLanguage.HA -> "Abu ko Dalilin Bashi *"
    }
    fun titlePlaceholder(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "ex: Perceuse Bosch, 50 €, Livre..."
        AppLanguage.EN -> "e.g., Drill, $50, Book..."
        AppLanguage.HA -> "mis: Na'ura, $50, Littafi..."
    }
    fun valueLabel(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Valeur ou Montant (Optionnel)"
        AppLanguage.EN -> "Value or Amount (Optional)"
        AppLanguage.HA -> "Darajar ko Adadi (Zabi)"
    }
    fun personHeader(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Personne concernée"
        AppLanguage.EN -> "Person involved"
        AppLanguage.HA -> "Mutumin da abin ya shafa"
    }
    fun contactNameLabel(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Nom du contact *"
        AppLanguage.EN -> "Contact name *"
        AppLanguage.HA -> "Sunan mutum *"
    }
    fun contactPhoneLabel(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Numéro de téléphone (Optionnel)"
        AppLanguage.EN -> "Phone number (Optional)"
        AppLanguage.HA -> "Lambar waya (Zabi)"
    }
    fun pickContactBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Contact"
        AppLanguage.EN -> "Contact"
        AppLanguage.HA -> "Lambar Waya"
    }
    fun mediaHeader(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Justificatifs & Médias joints"
        AppLanguage.EN -> "Attachments & Media"
        AppLanguage.HA -> "Hotuna & Fayilolin Bashi"
    }
    fun takePhotoBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Photo (Caméra)"
        AppLanguage.EN -> "Photo (Camera)"
        AppLanguage.HA -> "Hoto (Kamera)"
    }
    fun recordVideoBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Vidéo (Caméra)"
        AppLanguage.EN -> "Video (Camera)"
        AppLanguage.HA -> "Bidiyo (Kamera)"
    }
    fun galleryBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Galerie Médias"
        AppLanguage.EN -> "Media Gallery"
        AppLanguage.HA -> "Manhajar Hotuna"
    }
    fun voiceNoteHeader(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Note vocale"
        AppLanguage.EN -> "Voice note"
        AppLanguage.HA -> "Muryar Bayani"
    }
    fun recordBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Enregistrer"
        AppLanguage.EN -> "Record"
        AppLanguage.HA -> "Rikodi"
    }
    fun stopBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Arrêter"
        AppLanguage.EN -> "Stop"
        AppLanguage.HA -> "Tsaya"
    }
    fun reminderHeader(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Rappel de date de retour"
        AppLanguage.EN -> "Return Due Date Reminder"
        AppLanguage.HA -> "Ranar Tunatarwa ta Mayarwa"
    }
    fun setDateBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Définir date"
        AppLanguage.EN -> "Set date"
        AppLanguage.HA -> "Saka rana"
    }
    fun changeDateBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Changer"
        AppLanguage.EN -> "Change"
        AppLanguage.HA -> "Canza"
    }
    fun notesLabel(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Notes & détails complémentaires"
        AppLanguage.EN -> "Notes & additional details"
        AppLanguage.HA -> "Bayanai na daban"
    }
    fun saveBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Enregistrer le prêt"
        AppLanguage.EN -> "Save loan"
        AppLanguage.HA -> "Ajiye bashi"
    }

    // Detail Screen
    fun detailTitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Fiche du prêt"
        AppLanguage.EN -> "Loan Details"
        AppLanguage.HA -> "Bayanin Bashi"
    }
    fun lentTo(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Prêté à"
        AppLanguage.EN -> "Lent to"
        AppLanguage.HA -> "Aka bayar ga"
    }
    fun borrowedFrom(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Emprunté à"
        AppLanguage.EN -> "Borrowed from"
        AppLanguage.HA -> "Aka aro daga"
    }
    fun callBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Appeler"
        AppLanguage.EN -> "Call"
        AppLanguage.HA -> "Kira"
    }
    fun smsAction(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "SMS"
        AppLanguage.EN -> "SMS"
        AppLanguage.HA -> "Tura Saƙo"
    }
    fun whatsappAction(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "WhatsApp"
        AppLanguage.EN -> "WhatsApp"
        AppLanguage.HA -> "WhatsApp"
    }
    fun markReturnedBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Marquer comme RENDU"
        AppLanguage.EN -> "Mark as RETURNED"
        AppLanguage.HA -> "Shaida cewa AN MAYAR"
    }
    fun markActiveBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Marquer comme NON rendu"
        AppLanguage.EN -> "Mark as NOT returned"
        AppLanguage.HA -> "Shaida cewa BAI MAYAR BA"
    }
    fun deleteConfirmTitle(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Supprimer ce prêt ?"
        AppLanguage.EN -> "Delete this loan?"
        AppLanguage.HA -> "Goge wannan bashi?"
    }
    fun deleteConfirmDesc(lang: AppLanguage, title: String) = when(lang) {
        AppLanguage.FR -> "Êtes-vous sûr de vouloir supprimer la fiche de \"$title\" ? Action irréversible."
        AppLanguage.EN -> "Are you sure you want to delete \"$title\"? Action cannot be undone."
        AppLanguage.HA -> "Shin ka tabbata kana son goge \"$title\"? Ba za a iya soke shi ba."
    }
    fun cancelBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Annuler"
        AppLanguage.EN -> "Cancel"
        AppLanguage.HA -> "Fasa"
    }
    fun deleteBtn(lang: AppLanguage) = when(lang) {
        AppLanguage.FR -> "Supprimer"
        AppLanguage.EN -> "Delete"
        AppLanguage.HA -> "Goge"
    }
}

File: iosApp/PrendsEtRends/I18n/AppStrings.swift
import Foundation

public struct AppStrings {
    public static func appTitle(_ lang: AppLanguage) -> String { "Prends & Rends" }

    public static func appSubtitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Suivi des prêts & emprunts"
        case .en: return "Loan & borrow tracker"
        case .ha: return "Maikula da bashi da aro"
        }
    }

    // Tabs
    public static func tabAll(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Tous"
        case .en: return "All"
        case .ha: return "Dukkansu"
        }
    }
    public static func tabLent(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Prêtés"
        case .en: return "Lent"
        case .ha: return "Aka bayar"
        }
    }
    public static func tabBorrowed(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Empruntés"
        case .en: return "Borrowed"
        case .ha: return "Aka aro"
        }
    }

    // Status filters
    public static func filterAll(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Tous les statuts"
        case .en: return "All statuses"
        case .ha: return "Dukan matsayi"
        }
    }
    public static func filterActive(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "En cours"
        case .en: return "Active"
        case .ha: return "Yana gudana"
        }
    }
    public static func filterOverdue(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "En retard"
        case .en: return "Overdue"
        case .ha: return "Ya wuce lokaci"
        }
    }
    public static func filterReturned(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Rendu"
        case .en: return "Returned"
        case .ha: return "An mayar"
        }
    }

    // Dashboard
    public static func newLoanBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Nouveau prêt"
        case .en: return "New loan"
        case .ha: return "Sabon bashi"
        }
    }
    public static func searchPlaceholder(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Rechercher un objet, contact..."
        case .en: return "Search item, contact..."
        case .ha: return "Nemi abu, lamba..."
        }
    }
    public static func statsLent(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Prêtés"
        case .en: return "Lent"
        case .ha: return "Aka bayar"
        }
    }
    public static func statsBorrowed(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Empruntés"
        case .en: return "Borrowed"
        case .ha: return "Aka aro"
        }
    }
    public static func statsOverdue(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "En retard"
        case .en: return "Overdue"
        case .ha: return "Aka makara"
        }
    }
    public static func statsReturned(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Rendus"
        case .en: return "Returned"
        case .ha: return "Aka mayar"
        }
    }
    public static func emptyTitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Aucun prêt enregistré"
        case .en: return "No loans recorded"
        case .ha: return "Babu bashi a ajiye"
        }
    }
    public static func emptySubtitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Gardez une trace fluide de ce que vous prêtez ou empruntez."
        case .en: return "Keep smooth track of items and money lent or borrowed."
        case .ha: return "Kula da abin da kake bayarwa ko aro cikin sauki."
        }
    }

    // Form
    public static func createTitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Nouveau prêt"
        case .en: return "New Loan"
        case .ha: return "Sabon Bashi"
        }
    }
    public static func editTitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Modifier le prêt"
        case .en: return "Edit Loan"
        case .ha: return "Gyara Bashi"
        }
    }
    public static func typeHeader(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Type de prêt"
        case .en: return "Loan type"
        case .ha: return "Nau'in bashi"
        }
    }
    public static func typeLent(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "J'ai prêté"
        case .en: return "I lent"
        case .ha: return "Na bayar"
        }
    }
    public static func typeBorrowed(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "J'ai emprunté"
        case .en: return "I borrowed"
        case .ha: return "Na aro"
        }
    }
    public static func typeLentSub(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "À un proche"
        case .en: return "To a contact"
        case .ha: return "Ga wani"
        }
    }
    public static func typeBorrowedSub(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "D'un proche"
        case .en: return "From a contact"
        case .ha: return "Daga wani"
        }
    }
    public static func titleLabel(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Objet ou Motif du prêt *"
        case .en: return "Item or Reason *"
        case .ha: return "Abu ko Dalilin Bashi *"
        }
    }
    public static func titlePlaceholder(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "ex: Perceuse Bosch, 50 €, Livre..."
        case .en: return "e.g., Drill, $50, Book..."
        case .ha: return "mis: Na'ura, $50, Littafi..."
        }
    }
    public static func valueLabel(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Valeur ou Montant (Optionnel)"
        case .en: return "Value or Amount (Optional)"
        case .ha: return "Darajar ko Adadi (Zabi)"
        }
    }
    public static func personHeader(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Personne concernée"
        case .en: return "Person involved"
        case .ha: return "Mutumin da abin ya shafa"
        }
    }
    public static func contactNameLabel(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Nom du contact *"
        case .en: return "Contact name *"
        case .ha: return "Sunan mutum *"
        }
    }
    public static func contactPhoneLabel(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Numéro de téléphone (Optionnel)"
        case .en: return "Phone number (Optional)"
        case .ha: return "Lambar waya (Zabi)"
        }
    }
    public static func pickContactBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Contact"
        case .en: return "Contact"
        case .ha: return "Lambar Waya"
        }
    }
    public static func mediaHeader(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Justificatifs & Médias joints"
        case .en: return "Attachments & Media"
        case .ha: return "Hotuna & Fayilolin Bashi"
        }
    }
    public static func takePhotoBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Photo (Caméra)"
        case .en: return "Photo (Camera)"
        case .ha: return "Hoto (Kamera)"
        }
    }
    public static func recordVideoBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Vidéo (Caméra)"
        case .en: return "Video (Camera)"
        case .ha: return "Bidiyo (Kamera)"
        }
    }
    public static func galleryBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Galerie Médias"
        case .en: return "Media Gallery"
        case .ha: return "Manhajar Hotuna"
        }
    }
    public static func voiceNoteHeader(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Note vocale"
        case .en: return "Voice note"
        case .ha: return "Muryar Bayani"
        }
    }
    public static func recordBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Enregistrer"
        case .en: return "Record"
        case .ha: return "Rikodi"
        }
    }
    public static func stopBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Arrêter"
        case .en: return "Stop"
        case .ha: return "Tsaya"
        }
    }
    public static func reminderHeader(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Rappel de date de retour"
        case .en: return "Return Due Date Reminder"
        case .ha: return "Ranar Tunatarwa ta Mayarwa"
        }
    }
    public static func setDateBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Définir date"
        case .en: return "Set date"
        case .ha: return "Saka rana"
        }
    }
    public static func changeDateBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Changer"
        case .en: return "Change"
        case .ha: return "Canza"
        }
    }
    public static func notesLabel(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Notes & détails complémentaires"
        case .en: return "Notes & additional details"
        case .ha: return "Bayanai na daban"
        }
    }
    public static func saveBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Enregistrer le prêt"
        case .en: return "Save loan"
        case .ha: return "Ajiye bashi"
        }
    }

    // Detail Screen
    public static func detailTitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Fiche du prêt"
        case .en: return "Loan Details"
        case .ha: return "Bayanin Bashi"
        }
    }
    public static func lentTo(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Prêté à"
        case .en: return "Lent to"
        case .ha: return "Aka bayar ga"
        }
    }
    public static func borrowedFrom(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Emprunté à"
        case .en: return "Borrowed from"
        case .ha: return "Aka aro daga"
        }
    }
    public static func callBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Appeler"
        case .en: return "Call"
        case .ha: return "Kira"
        }
    }
    public static func smsAction(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "SMS"
        case .en: return "SMS"
        case .ha: return "Tura Saƙo"
        }
    }
    public static func whatsappAction(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "WhatsApp"
        case .en: return "WhatsApp"
        case .ha: return "WhatsApp"
        }
    }
    public static func markReturnedBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Marquer comme RENDU"
        case .en: return "Mark as RETURNED"
        case .ha: return "Shaida cewa AN MAYAR"
        }
    }
    public static func markActiveBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Marquer comme NON rendu"
        case .en: return "Mark as NOT returned"
        case .ha: return "Shaida cewa BAI MAYAR BA"
        }
    }
    public static func deleteConfirmTitle(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Supprimer ce prêt ?"
        case .en: return "Delete this loan?"
        case .ha: return "Goge wannan bashi?"
        }
    }
    public static func deleteConfirmDesc(_ lang: AppLanguage, title: String) -> String {
        switch lang {
        case .fr: return "Êtes-vous sûr de vouloir supprimer la fiche de \"\(title)\" ? Action irréversible."
        case .en: return "Are you sure you want to delete \"\(title)\"? Action cannot be undone."
        case .ha: return "Shin ka tabbata kana son goge \"\(title)\"? Ba za a iya soke shi ba."
        }
    }
    public static func cancelBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Annuler"
        case .en: return "Cancel"
        case .ha: return "Fasa"
        }
    }
    public static func deleteBtn(_ lang: AppLanguage) -> String {
        switch lang {
        case .fr: return "Supprimer"
        case .en: return "Delete"
        case .ha: return "Goge"
        }
    }
}
